src/uct/ib/rc/accel/rc_mlx5_common.c

```diff
@@ -206,7 +206,7 @@ unsigned uct_rc_mlx5_iface_srq_post_recv_ll(uct_rc_mlx5_iface_common_t *iface)
     return count;
 }
 
-void uct_rc_mlx5_iface_common_prepost_recvs(uct_rc_mlx5_iface_common_t *iface)
+void uct_rc_mlx5_common_prepost_recvs(uct_rc_mlx5_iface_common_t *iface)
```

Comment:
imo it is ok to leave the name as it is, because preposting itself happens on `iface`, not `ep`

Reply:
Yes, but file name is usually a prefix, that is why I changed the name. And the involvement of the interface is reflected in the parameters.

Reply:
There are the following options:
1. leave it as-is (`uct_rc_mlx5_iface_common_prepost_recvs(iface)`), because many function names started with `uct_rc_mlx5_iface` in this header file.
2. change name to `uct_rc_mlx5_common_prepost_recvs_on_iface(iface)`, or `uct_rc_mlx5_common_iface_prepost_recvs(iface)`, to have `iface` in the name and to use more suitable prefix.
3. use `uct_rc_mlx5_common_prepost_recvs(iface)`, because I think there is a word duplication in the second item.

I'd prefer the third. @brminich would you still vote for the first?

Reply:
according to our naming convention, we use the object as a prefix which is  `uct_rc_mlx5_iface`, so i'd not change the name. 
the second option does not look consistent with the rest of the code.
3-d is better than the 2d, but the 1st one is still preferrable for me (also because it's better not to change just the name to avoid messing up commits history)